### PR TITLE
[_]: Feat/banner for lifetime campaign

### PR DESCRIPTION
**Pull request by xabg2**
* New banner for the lifetime campaign. 
* Fixed a small problem with the language box.

### CODE REVIEW

File: src/app/banners/Banner.tsx [hunk 1]
```diff
@@ -1,11 +1,10 @@
 import React, { useEffect } from 'react';
 import { X } from 'phosphor-react';
-import Hearts from 'assets/images/banner/hearts.png';
-import NeonBlur from 'assets/images/banner/neonBlur.png';
+import NeonBlur from 'assets/images/banner/infinity.svg';
 import { useTranslationContext } from 'app/i18n/provider/TranslationProvider';
 
 const Banner = () => {
-  const [showLifetimeBanner, setShowLifetimeBanner] = React.useState(false);
+  const [showLifetimeBanner, setShowLifetimeBanner] = React.useState(true);
```
Comment (CandelR) [inline src/app/banners/Banner.tsx]: I would add the opening and closing logic and the possible variables outside the Banner component, so you can reuse it for each banner you need for the different campaigns by creating a wrapper.

File: src/app/core/views/Preferences/tabs/Account/Language.tsx [hunk 2]
```diff
@@ -25,8 +25,8 @@ export default function Language(): JSX.Element {
 
   useEffect(() => {
     if (localStorageLanguage) {
-      setCurrentLangText(currentLang[localStorageLanguage as string]);
-      setLang(localStorageLanguage as string);
+      setCurrentLangText(currentLang[localStorageLanguage]);
+      setLang(localStorageLanguage);
```
Comment (CandelR) [inline src/app/core/views/Preferences/tabs/Account/Language.tsx]: If you always have to change both states at the same time, add it to a function, I see that the code is repeated 3 times.

File: src/app/drive/components/DriveExplorer/DriveExplorer.tsx [hunk 3]
```diff
@@ -155,24 +154,10 @@ const DriveExplorer = (props: DriveExplorerProps): JSX.Element => {
...
-    if (!isSignUpTutorialCompleted && currentTutorialStep === 1 && successNotifications.length > 0) {
-      setShowSecondTutorialStep(true);
-    }
-  }, [successNotifications]);
+  const plan = useSelector<RootState, PlanState>((state) => state.plan);
 
   const showTutorial =
-    useAppSelector(userSelectors.hasSignedToday) &&
-    !isSignUpTutorialCompleted &&
-    (showSecondTutorialStep || currentTutorialStep === 0);
```
Comment (CandelR) [inline src/app/drive/components/DriveExplorer/DriveExplorer.tsx]: Why did you remove it? This will break the onboarding tutorial

**Review by CandelR — CHANGES_REQUESTED** [review]

File: src/app/drive/components/DriveExplorer/DriveExplorer.tsx (hunk 3 above)
Comment (xabg2) [reply src/app/drive/components/DriveExplorer/DriveExplorer.tsx]: Ups. It is my fault

**Review by CandelR — APPROVED** [review]
